lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -49,9 +51,24 @@ def refresh
         end
       end
 
-      def close
+      def timeout
+        logger.debug('job timed out')
         super
-        FileUtils.remove_entry(@root) if @tmp_working_dir
```

Comment:
Why are we removing this? :thinking:
It was added recently to clean-up the filesystem after ansible run.

Reply:
@ezr-ondrej You are right I added it again.